### ISSUE TITLE
.ipynb colour showing up in title

### ISSUE THREAD

**Issue by widdowquinn** [LiquidFun/aoc_tiles#8]
Hi,

Thank you for making this. I'm finding a wee niggle though in the rendered title.

I'm doing AoC in Jupyter notebooks, and the titls is being rendered with the hexcode for the colour defined in [`colors.py`](https://github.com/LiquidFun/aoc_tiles/blob/main/aoc_tiles/colors.py)

<img width="415" alt="Screenshot 2024-12-13 at 17 32 11" src="https://github.com/user-attachments/assets/f329c888-8864-469e-bbc1-04b01a46e744" />

The HTML generated looks like this:

```html
<!-- AOC TILES BEGIN -->
<h1 align="center">
  2024 - 26 ⭐ - #DA5B0B
</h1>
<a href="day01.ipynb">
[...]
```

I'm assuming that's not meant to happen and might be a result of the way the colours are extended from `github_languages.yaml`. At least, I've not spotted a flag/option I can use to change it, yet. If there's something I can do to fix it I'd be happy to know about it.

Cheers,

L.

**Comment by LiquidFun** [comment]
That's funny :D. Yeah, you are right the include was quite naive and was also being used as the name of the language. I've improved the override -> should be fixed in [0.6.3](https://github.com/LiquidFun/aoc_tiles/releases/tag/0.6.3)

**Comment by widdowquinn** [comment]
Many thanks, @LiquidFun - that's all fixed at this end now, with the update to 0.6.3!

Much appreciated!
